#!/bin/sh -e
##  by Joost Yervante Damad <joost@lumatec.be>
##
## DP: verbose probing

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -ur openmsx-0.5.2.orig/build/probe.mk openmsx-0.5.2/build/probe.mk
--- openmsx-0.5.2.orig/build/probe.mk        2004-10-15 01:00:34.000000000 +0200
+++ openmsx-0.5.2/build/probe.mk    2005-07-04 19:03:20.000000000 +0200
@@ -20,7 +20,7 @@
 endif

 # Result files.
-LOG:=$(OUTDIR)/probe.log
+LOG:=/dev/stdout
 OUTHEADER:=$(OUTDIR)/probed_defs.hh
 OUTMAKE:=$(OUTDIR)/probed_defs.mk

